client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -22,6 +40,28 @@ const NewStatsNotices = ( { siteId, isOdysseyStats }: StatsNoticesProps ) => {
 		isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
 	);
 
+	const reduxDispatch = useDispatch();
+	// TODO: Display error messages on the notice.
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	const [ error, setError ] = useState< Error | null >( null );
+
+	// Update site purchases in Redux store for Odyssey Stats.
+	useEffect( () => {
```

Comment:
should we wait for the request to finish before showing notices? otherwise the notice might flicker I guess. 🤔

Reply:
Since the sync latency only occurs when removing the subscription, the notice would not be flicking from hidden to shown.

Actually, there is a flicker only on Calypso Stats when the purchase is made. 😅

https://github.com/Automattic/wp-calypso/assets/6869813/7d3716b9-eb3e-4012-bce5-1da42cd8b580